flink-table/flink-table-common/src/main/java/org/apache/flink/table/utils/DateTimeUtils.java

```diff
@@ -1646,6 +1646,13 @@ public static int addMonths(int date, int m) {
         return ymdToUnixDate(y0, m0, d0);
     }
 
+    /** Adds a given number of months to a dateStr, represented as the date since the epoch. */
+    public static String addMonths(String dateStr, int m) {
+        int date1 = DateTimeUtils.parseDate(dateStr, DATE_FORMAT_STRING);
```

Comment:
Do we really need to have `DateTimeUtils.parseDate` if we are already in `DateTimeUtils` class?